EXILED/Exiled.API/Extensions/MirrorExtensions.cs

```diff
@@ -289,6 +289,10 @@ public static void ChangeAppearance(this Player player, RoleTypeId type, IEnumer
                     isRisky = true;
 
                 writer.WriteUShort((ushort)Mathf.Clamp(Mathf.CeilToInt(player.MaxHealth), ushort.MinValue, ushort.MaxValue));
+                writer.WriteBool(true);
+                ushort value = 0;
+                writer.WriteRelativePosition(player.RelativePosition);
+                writer.WriteUShort(value);
```

Comment:
```suggestion
```

Reply:
instead put the ZombieRole before the FpcStandardRoleBase

Reply:
![image](https://github.com/user-attachments/assets/045a1a44-bfce-4ab7-b28b-e8d7a3158207)

Reply:
![image](https://github.com/user-attachments/assets/3fd21c41-a6ba-42dc-892b-10926dc4b341)

Your suggestion is wrong as far as I can tell

Reply:
Please describe in more detail what exactly you want/disagree with

Reply:
I see, it's actually both - sigh, I understand

Reply:
Also, it probably worked because of the missing data in general - if I had to guess it needed 40 bytes (example) and it had 35, so giving it 60 probably didn't matter because it only reads in the first 40. (AKA giving it the extra data after the previous condition gives the spawn data probably didn't break because it reads only a certain amount of data)